Prog/Cvd19_affctd_ind_pop_tables.sas

```diff
@@ -0,0 +1,678 @@
+    
+  value hispan_sum (notsorted)
+    416 = 'Salvadoran'
+    401-415, 417 = 'Other Central American'
+    100-300, 420-499 = 'Other Latino/Hispanic origin'
+    0 = 'Not Latino/Hispanic';
+    
+  value poverty_sum (notsorted)
+    1-100 = 'At or below poverty'
+    100<-200 = '101 to 200% poverty'
+    200<-high = 'Above 200% poverty'
+    0 = 'Poverty status not determined';
+    
+  value educ_sum (notsorted)
+    0 - 61 = 'Less than HS diploma'
+    62 - 71 = 'HS diploma/GED'
+    81 = 'Associates degree'
+    101 = 'Bachelors degree'
+    114-high = 'Masters degree or higher';
+    
+  value hsg_cost_ratio (notsorted)
+    0.50 - high = 'Severe cost burden (50%+ income)'
+    0.30 -< 0.50 = 'Cost burden (30 - 49%)'
+    0 -< 0.30 = 'No cost burden (< 30%)'
+    .n = 'n/a';
+    
+  value yesnona (notsorted)
+    1 = 'Yes'
+    0 = 'No'
+    .n = 'n/a';
+    
+  value OWNERSHP_f (notsorted)
+    1 = "Owned or being bought (loan)"
+    2 = "Rented"
+    0 = "n/a"
+  ;
+    
+  /** HUD Income Categories **/
+  value hudinc (notsorted)
+    1 = 'Extremely low (0-30% AMI)'
+    2 = 'Very low (31-50%)'
+    3 = 'Low (51-80%)'
+    4 = 'Middle (81-120%)'
+    5 = 'High (over 120%)'
+    .n = 'n/a';    
+    
+  value hudinc_low (notsorted)
+    1-3 = 'At or below low income (0-80% AMI)'
+    4-5 = 'Above low income (81% AMI and higher)'
+    .n = 'n/a';    
+    
+  value $upuma_to_mwcog_jurisd (notsorted) 
+    "5159301", "5159302", "5159303", "5159304", "5159305", "5159306", "5159307", "5159308", "5159309" = "Fairfax Co., Fairfax city, and Falls Church"
+    "2401101", "2401102", "2401103", "2401104", "2401105", "2401106", "2401107" = "Prince George's County"
+    "2401001", "2401002", "2401003", "2401004", "2401005", "2401006", "2401007" = "Montgomery County"
+    "1100101", "1100102", "1100103", "1100104", "1100105" = "District of Columbia"
+    "5151244", "5151245", "5151246" = "Prince William Co., Manassas, and Manassas Park" 
+    "5110701", "5110702" , "5110703" = "Loudoun County"
+    "2400301", "2400302" = "Frederick County"
+    "5101301", "5101302" = "Arlington County"
+    "5151255" = "Alexandria"
+    "2401600" = "Charles County"
+    other = "Not in MWCOG region";
+    
+  value race_ethn (notsorted)
+     1 = "White non-Latino"
+     2 = "Black non-Latino"
+     3 = "Latino"
+     4 = "Asian non-Latino"
+     5 = "All other non-Latino"
+    .n = "Not available";
+    
+  value cvd19_affctd_ind (notsorted) 
+    1 = 'In COVID-19 affected industry'
+    0 = 'Not in COVID-19 affected industry'
+    other = 'n/a';
+    
+  value pct_inc_split (notsorted)
+    40 <- high = 'Above 40% household income'
+    0 - 40 = 'At or below 40% household income'
+    other = 'n/a';
+    
+run;  
+
+** Examine earnings for outliers **;
+
+proc univariate data=Covid19.cvd19_affctd_ind_pop;
+  where mwcog_region = 1 and cvd19_affctd_ind = 1 and incearn > 0;
+  var incearn;
+run;
+
+%let CVD19_TOP_1PCT_EARNINGS = 235658;
```

Comment:
It's probably not worth doing since this won't be a recurring program but you can use proc SQL to pull the top and bottom percent into a macro automatically.